oss-cloud-backend/gitHubApiService.js

```diff
@@ -48,35 +45,35 @@ module.exports.getParentRepos = async (repos) => {
 }
 
 // retrieves pull requests for given user for each repo in given array
-module.exports.getUserPulls = async (username, repos) => {
-  let pulls = [];
-  const pullsPromises = repos.map(async (repo) => {
-    let repoPulls = await searchUserPulls(username, repo);
-    pulls = pulls.concat(repoPulls);
-    return repoPulls;
+module.exports.getUserPullRequests = async (username, repos) => {
+  let pullRequests = [];
+  const pullRequestPromises = repos.map(async (repo) => {
+    let repoPullRequests = await searchUserPullRequests(username, repo);
+    pullRequests = pullRequests.concat(repoPullRequests);
+    return repoPullRequests;
   })
 
-  await Promise.all(pullsPromises);
+  await Promise.all(pullRequestPromises);
 
-  console.log("Retrieved", pulls.length, "for user", username)
-  return pulls;
+  console.log("Retrieved", pullRequests.length, "for user", username)
+  return pullRequests;
 }
 
 // returns pull requests from a repo where given user is the author
-searchUserPulls = async(username, repo) => {
+searchUserPullRequests = async(username, repo) => {
   console.log("searching pull requests: ", username, repo.name)
-  var pulls = []
-  let pullNum = 0;
+  var pullRequests = []
+  let pullRequestNum = 0;
   do {
     await octokit.search.issuesAndPullRequests({
       q: "repo:" + repo.owner.login + "/" + repo.name + "+author:" + username + "+is:pr",
       per_page: 100

```

Comment:
`page` parameter is missing here (https://octokit.github.io/rest.js/#octokit-routes-search-issues-and-pull-requests)

Reply:
Added page parameter.